api/src/main/java/net/kyori/adventure/text/TranslatableComponent.java

```diff
@@ -119,12 +120,35 @@ public interface TranslatableComponent extends BuildableComponent<TranslatableCo
   @Contract(pure = true)
   @NotNull TranslatableComponent args(final @NotNull List<? extends ComponentLike> args);
 
+  /**
+   * Gets the translation fallback text for this component.
+   * The fallback text will be shown when the client doesn't know the
+   * translation key used in the translatable component.
+   *
+   * @return the fallback string
+   * @since 4.13.0
+   */
+  @Nullable String fallback();
```

Comment:
p much everything relating to this fallback feature needs an `@sinceMinecraft` annotation